pyaerocom/io/read_airnow.py

```diff
@@ -395,15 +442,82 @@ def _filedata_to_statlist(self, arrs, vars_to_retrieve):
                 stats.append(stat)
         return stats
 
-    def read_file(self):
+    def read_file(self, filename, vars_to_retrieve=None):
```

Comment:
Could use type hints

Reply:
The whole reader is without type hints. I don't have much time to burn for this right now...